tnt-persistence/src/main/java/org/dcsa/tnt/persistence/entity/Reference.java

```diff
@@ -31,15 +23,17 @@ public class Reference {
   @Column(name = "reference_value", length = 100, nullable = false)
   private String referenceValue;
 
-  @Column(name = "shipment_id")
-  private UUID shipmentID;
+  @Column(name = "utilized_transport_equipment_id")
+  private UUID utilizedEquipmentID;
 
-  @Column(name = "shipping_instruction_id")
-  private UUID shippingInstructionID;
+  @Column(name = "document_id")
+  private UUID documentID;
 
-  @Column(name = "booking_id")
-  private UUID bookingID;
+  @Column(name = "transport_call_id")
+  private UUID transportCallID;
 
-  @Column(name = "consignment_item_id")
-  private UUID consignmentItemID;
+  @Id // This is not a real ID (just a workaround) - Done to satisfy JPA Entity ID requirement
+  // Note that this is a VIEW -- NOT a schema
+  @Column(name = "link_type")
+  private String linkType;
```

Comment:
I thought you concluded this would **not** work?

Reply:
Yes, it will not. Now fixed